ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/individual-journey.ts

```diff
@@ -144,8 +158,38 @@ export class IndividualJourney extends JourneyBase {
     }
   }
 
-  private async submit() {
-    // call the save service
+  private async submit(model: MutableIndividualSuitabilityModel) {
+    const mapper = new IndividualModelMapper();
+    let answers: HearingSuitabilityAnswer[];
+    answers = mapper.mapToRequest(model);
```

Comment:
could you hide all of this so that the journey doesn't have to know about requests and apis?

Reply:
Moved this to a submit service.